controllers/suite_test.go

```diff
@@ -77,8 +78,8 @@ var (
 
 	namespaceDeletionSupported bool
 
-	timeout  = time.Second * 10
-	interval = time.Millisecond * 10
+	timeout  = time.Second * 1
+	interval = time.Millisecond * 50
```

Comment:
If you do change the baseDelay to 10ms above then you can have the interval to be 10ms here.